includes/widgets/image-carousel.php

```diff
@@ -647,6 +649,48 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'dots_space',
+			[
+				'label' => esc_html__( 'Space Between Dots', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'separator' => 'after',
+				'size_units' => [ 'px', 'em', 'rem', '%', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}} .swiper-pagination-bullet' => 'margin-left: {{SIZE}}{{UNIT}};',
```

Comment:
@arielk, should I add a CSS selector '{{WRAPPER}} .swiper-pagination-bullet:not(:first-child)' so that the spacing does not apply to the 1st child element?